src/runtime.js

```diff
@@ -172,46 +172,57 @@ function adoptShared(global, callback) {
   /** @const */
   global.AMP.toggleExperiment = toggleExperiment.bind(null, global);
 
+  /**
+   * Sets the function to forward tick events to.
+   * @param {function(string,?string=,number=)} fn
+   * @param {function()=} opt_flush
+   * @deprecated
+   * @export
+   */
+  global.AMP.setTickFunction = (fn, opt_flush) => {};
```

Comment:
we should be able to remove this now, as the viewer no longer uses this.

cc @ericfs